torchvision/datasets/lsun.py

```diff
@@ -69,13 +70,17 @@ class LSUN(data.Dataset):
             target and transforms it.
     """
 
-    def __init__(self, db_path, classes='train',
+    def __init__(self, root, classes='train',
                  transform=None, target_transform=None):
         categories = ['bedroom', 'bridge', 'church_outdoor', 'classroom',
                       'conference_room', 'dining_room', 'kitchen',
                       'living_room', 'restaurant', 'tower']
         dset_opts = ['train', 'val', 'test']
-        self.db_path = db_path
+        self.root = os.path.expanduser(root)
+        self.transform = transform
```

Comment:
`transform` is not used in `self`, but it is passed to `LSUNClass` so this is unnecessary.

Reply:
oh, looks like this is for `__repr__`. Sounds good then